src/jit-dasm-pmi/jit-dasm-pmi.cs

```diff
@@ -147,6 +149,7 @@ private void Validate()
         public IReadOnlyList<string> PlatformPaths { get { return _platformPaths; } }
         public string FileName { get { return _fileName; } }
         public IReadOnlyList<string> AssemblyList { get { return _assemblyList; } }
+        public bool Tiering => _tiering;
```

Comment:
Doesn't this syntax make the property read/write, when all we really want is to make it readable?

Reply:
No, this is just a getter, to be able to set you'd need something like:
```
   public Bool Tiering
   {
      get => _tiering;
      set => _tiering = value;
   }
```